src/contract.rs

```diff
@@ -358,10 +362,42 @@ fn validate_fee_params_get_messages(
 ///
 /// MIGRATE SECTION
 ///
-/// TODO: Allow fee amount swap across migrations
-///
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    Ok(Response::default())
+#[entry_point]
+pub fn migrate(
```

Comment:
Beyond the 1.0.0-beta changes, I added a few tweaks to allow us to optionally change fee stuff when migrating